packages/v4/patternfly-docs/pages/componentGallery.js

```diff
@@ -0,0 +1,139 @@
+  const drawerRef = React.useRef();
+  // convert summary text in drawer from string to jsx
+  const SummaryComponent = ({ id }) => {
+    const componentDasherized = id.split(' ').join('-').toLowerCase();
+    const summary = componentsData?.[componentDasherized]?.summary;
+    if (!summary) {
+      return null;
+    }
+    const { code } = convertToReactComponent(`<>${summary}</>`);
+    const getSummaryComponent = new Function('React', 'Link', code);
+    return getSummaryComponent(React, Link);
+  }
+
+  const onChange = (labelledById, _event) => {
+    const newSelectedCard = labelledById === selectedCard ? null : labelledById;
+    setSelectedCard(newSelectedCard);
+  };
+  
+  const onClick = (componentData) => {
+    if (selectedCard !== componentData.id) {
+      setSelectedCard(componentData.id);
+      setDrawerPanelData(componentData);
+      setIsExpanded(true);
+    } else {
+      setIsExpanded(!isExpanded);
+    }
+  };
+  const onKeyDown = (event, componentData) => {
+    if (event.target !== event.currentTarget) {
+      return;
+    }
+    if ([' ', 'Enter'].includes(event.key)) {
+      event.preventDefault();
+      onClick(componentData);
+    }
+  };
+
+  return (
+    <div className="ws-component-gallery">
+      <Toolbar isSticky>
+        <ToolbarContent>
+          <ToolbarGroup alignment={{default: 'alignLeft'}}>
+            <ToolbarItem widths={{default: '320px'}}>
+              <SearchInput value={searchTerm} placeholder="Search components by name" onChange={setSearchTerm} />
+            </ToolbarItem>
+          </ToolbarGroup>
+          <ToolbarGroup>
+            <ToolbarItem alignment={{default: 'alignLeft'}}>
+              <Button variant="link" onClick={() => setSearchTerm('')} isInline>Reset</Button>
+            </ToolbarItem>
+          </ToolbarGroup>
+          <ToolbarGroup alignment={{default: 'alignRight'}} spacer={{default: 'spacerMd', md: 'spacerNone'}}>
+            <ToolbarItem alignment={{default: 'alignRight'}}>
+              <Text component={TextVariants.small}>{filteredComponents.length} items</Text>
+            </ToolbarItem>
+          </ToolbarGroup>
+        </ToolbarContent>
+      </Toolbar>
+      <Sidebar isPanelRight className='ws-component-gallery' tabIndex={1} orientation={() => (window.innerWidth < 768 ? 'stack' : 'split')}>
+        <SidebarPanel variant="sticky">
+          <TextContent>
+            <Text component={TextVariants.h2}>
+              <span tabIndex={isExpanded ? 0 : -1} ref={drawerRef}>
+                {drawerPanelData.title}
+              </span>
+            </Text>
+            <Text>
+              { drawerPanelData.id ? <SummaryComponent id={drawerPanelData.id}/> : null }
+            </Text>
+          </TextContent>
+          <Button className="ws-view-component" component={(props) => <Link {...props} to={drawerPanelData.slug} />}>View component</Button>
+        </SidebarPanel>
+        <SidebarContent>
```

Comment:
this should get rid of the white background behind the cards

```suggestion
        <SidebarContent hasNoBackground >
```